### PR TITLE
test(navmenu): make sure when click menu item class will change to active state

### LINKED ISSUE #1456: [NV-929] - Add cypress component test to NavMenu component
## Why?

To help increase our test coverage within the `novu/web` project, we can add the newly released [cypress component tests](https://docs.cypress.io/guides/component-testing/testing-react).

## How?

A new component test file should be added located near the original source file(see example at `apps/web/src/design-system/button/button.cy.tsx` and could be run after running `npm run start` from the root of the monorepo and specifying "Run Tests" and "Open Cypress", after which select the "Component testing" in the cypress screen.

Write the new test and make sure it passes all assertions.

The component is located at: apps/web/src/design-system/navigation/NavMenu.tsx

### PR DESCRIPTION
**Pull request by chaiwattsw**
closes #1456

- **What kind of change does this PR introduce?** 
Cypress component test

- **Why was this change needed?** 
#1456

- **Other information**:

### CODE REVIEW

**Comment by chaiwattsw** [comment]
@scopsy Can you please review this PR

File: apps/web/src/design-system/navigation/NavMenu.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { NavMenu } from './NavMenu';
+import { MemoryRouter as Router } from 'react-router-dom';
+import { TestWrapper } from '../../testing';
+
+it('should active when clicked menu item', () => {
```
Comment (p-fernandez) [inline apps/web/src/design-system/navigation/NavMenu.cy.tsx:5]:
```suggestion
it('should have active class when clicked menu item', () => {
```

File: apps/web/src/design-system/navigation/NavMenu.cy.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  const activeClass = 'mantine-g10fyh';
+  const notActiveClass = 'mantine-16w7c2e';
```
Comment (djabarovgeorge) [inline apps/web/src/design-system/navigation/NavMenu.cy.tsx:7]: Hey love the new component test, do mantine create those classes always with the same class name as above?

File: apps/web/src/design-system/navigation/NavMenu.cy.tsx (hunk 1 above)
Comment (p-fernandez) [inline apps/web/src/design-system/navigation/NavMenu.cy.tsx:5]: Please also add a describe for the test suite surrounding the test you implemented.
```
describe('NavMenu', () => {
...
});
```

**Review by p-fernandez — APPROVED** [review]
🌟

File: apps/web/src/design-system/navigation/NavMenu.cy.tsx (hunk 2 above)
Comment (chaiwattsw) [reply apps/web/src/design-system/navigation/NavMenu.cy.tsx:7]: HI @djabarovgeorge Yes. always the same. like styled component

**Review by p-fernandez — APPROVED** [review]

**Pull request merged**